.cpyskip.txt

```diff
@@ -117,3 +118,4 @@ tools/unicodetools/*
 # vendor
 #
 vendor/double-conversion/upstream/*
+vendor/json/upstream/*
```

Comment:
will it need to also skip the file in its copied in location? might be needed

Reply:
I changed the copied-in `json-json.hpp` file to include the Unicode license at the top, and the pull-from-upstream script should preserve the change, so I think that's OK? Though I'll look at the failing copyright-scan output.

Reply:
it looked like that was fine, just the readme